src/Glpi/Form/Comment.php

```diff
@@ -36,15 +36,21 @@
 namespace Glpi\Form;
 
 use CommonDBChild;
+use CommonDBTM;
 use Glpi\Application\View\TemplateRenderer;
+use Glpi\Form\Translation\Context\FormTranslationHandler;
+use Glpi\Form\Translation\Context\ProvideFormTranslationsInterface;
 use Log;
 use Override;
 
 /**
  * Comment of a given helpdesk form's section
  */
-final class Comment extends CommonDBChild implements BlockInterface
+final class Comment extends CommonDBChild implements BlockInterface, ProvideFormTranslationsInterface
```

Comment:
```suggestion
final class Comment extends CommonDBChild implements BlockInterface
```

`ProvideFormTranslationsInterface` is included in `BlockInterface`.